### PR TITLE
Letters App Content Updates

### PR DESCRIPTION
**Pull request by markgreenburg**
This PR includes a few light content changes:
- Text update for the service letter description and warning
- Change `you are` in the partner service down error message to `you‘re`
- Text update to 'need help' phone number in `LetterList`; now points to VA Benefits Help Desk

Issues:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5659
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5665
https://github.com/department-of-veterans-affairs/vets-website/pull/6696

### CODE REVIEW

**Comment by markgreenburg** [comment]
Resolves https://github.com/department-of-veterans-affairs/vets.gov-team/issues/5665 as well

**Comment by rroueche** [comment]
@markgreenburg since I can't spin this up locally, could you provide screenshots here of the changes? I'll reference the code and the screenshots to approve!

File: src/js/letters/containers/LetterList.jsx
Comment (cvalarida) [inline src/js/letters/containers/LetterList.jsx]: Wrong curly quote. I think the one you want to use is `’`. Then again, it could be `’`... :man_shrugging: Unicode is hard.

**Review by cvalarida — COMMENTED** [review]
👍 
I'll let @rroueche approve after you get the screenshots, though.

**Comment by markgreenburg** [comment]
Sure thing:

Service Verification Letter Warning:
screen shot 2017-10-31 at 1 12 09 pm

Partner Service Down & Help Phone Change:
screen shot 2017-10-31 at 1 13 48 pm

File: src/js/letters/containers/LetterList.jsx
Comment (rroueche) [reply src/js/letters/containers/LetterList.jsx]: ^ agree re:the type of curly quote it should be

Comment (rroueche) [reply src/js/letters/containers/LetterList.jsx]: I mean, it needs to be updated to the correct apostrophe symbol :)

Comment (markgreenburg) [reply src/js/letters/containers/LetterList.jsx]: Ugh, yep. ' vs. ‘ vs. ’ - had the second and needed the third. I seem to always forget that it's still an opening curly even though the shortcut uses a closing bracket. 👎

**Review by rroueche — APPROVED** [review]
lgtm! pursuant to the apostrophe update!

**Pull request merged**
